tests/SideBySide.New/AppConfig.cs

```diff
@@ -20,8 +20,8 @@ public static class AppConfig
 		private static int _configFirst;
 
 		private static IConfiguration ConfigBuilder { get; } = new ConfigurationBuilder()
-			.SetBasePath(File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "config.json")) ? Directory.GetCurrentDirectory() :
-				Path.Combine(Directory.GetCurrentDirectory(), "tests", "SideBySide.New"))
+			.SetBasePath(Path.GetFileName(Path.GetDirectoryName(Path.Combine(Directory.GetCurrentDirectory(), "config.json"))) == "SideBySide.New"
```

Comment:
```
Path.GetDirectoryName(Path.Combine(Directory.GetCurrentDirectory(), "config.json"))
```

seems like a complicated way of writing `Directory.GetCurrentDirectory()` (i.e., adding and removing `\config.json` from the path).

Additionally, this breaks the Baseline tests, which are looking for the tests in the current working directory (set by `xunit.runner`) not in `SideBySide.New`. Was there a reason this needed to be changed?